Mathlib/Topology/Constructions.lean

```diff
@@ -1089,17 +1089,6 @@ theorem IsOpenMap.restrict {f : X → Y} (hf : IsOpenMap f) {s : Set X} (hs : Is
   hf.comp hs.isOpenMap_subtype_val
 #align is_open_map.restrict IsOpenMap.restrict
 
-lemma IsClosedMap.restrictPreimage {f : X → Y} (hcl : IsClosedMap f) (T : Set Y) :
```

Comment:
Is this lemma now deleted? Could you add it as (deprecated) alias of `Set.restrictPreimage_isClosedMap`? Arguably it shouldn't even be deprecated, because this name allows to use dot notation.

Reply:
The point is there was a duplicate, but maybe we want to rename the other version of the lemma to use dot notation.

Reply:
Please do add a deprecation, something like
```
@[deprecated] -- Since <date>
alias oldName := newName
```
(I personally like this name better than the other one, so maybe you can rename the other one to `IsClosedMap.restrictPreimage`?)

Reply:
I added a deprecation for `Set.restrictPreimage_isClosedMap` and `Set.restrictPreimage_isOpenMap`, so the true names are now `IsClosedMap.restrictPreimage` and `IsOpenMap.restrictPreimage` (with the right order for the arguments).